src/prolly/leaf.ts

```diff
@@ -14,9 +17,16 @@ export class Leaf {
 
   static load(chunk: Chunk): Leaf {
     // Validate at load-time so we can assume data is valid thereafter.
-    const entries = chunk.data;
-    // Assert that the shape/type is correct
-    assertEntries(entries);
+    let entries: Entry[];
+    const data = chunk.data;
+    if (data instanceof Uint8Array) {
```

Comment:
hm. this is scary but i guess makes sense. Can we please add a comment like:

```
older leaf data was flat buffer encoded
```

Reply:
Oh, I forgot to add the `READ_FLATBUFFER` const here